addons/website_hr_recruitment/models/hr_recruitment.py

```diff
@@ -59,3 +58,71 @@ def _compute_website_url(self):
     def set_open(self):
         self.write({'website_published': False})
         return super(Job, self).set_open()
+
+
+class Country(models.Model):
+    _inherit = 'res.country'
+
+    has_job = fields.Boolean(compute='_compute_has_job', search='_search_has_job')
+
+    def _compute_has_job(self):
+        for country in self:
+            jobs = self.env['hr.job'].sudo().search(
+                [('website_published', '=', True), ('no_of_recruitment', '>', 0), ('address_id.country_id', '=', country.id)]
+            )
+            country.has_job = bool(len(jobs))
+
+    def _search_has_job(self, operator, value):
+        if operator not in ('=', '!=', '<>'):
+            raise ValueError('Invalid operator: %s' % (operator,))
```

Comment:
The operator is not technically _Invalid_, but probably just _Unsupported_. We could also just return an empty result instead of raising, not sure...